jobs/config.json

```diff
@@ -11477,6 +11477,21 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-local-e2e-containerized": {
+    "args": [
+      "--deployment=local",
+      "--extract=ci/latest",
+      "--extract-source",
+      "--ginkgo-parallel=1",
+      "--provider=local",
+      "--test_args=--ginkgo.focus=\\[Conformance\\]",
+      "--timeout=120m"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-node"
```

Comment:
hummm, sig-conformance?

Reply:
do we have a `sig-conformance`? i can't find it https://k8s-code.appspot.com/?q=sig-conformance&i=nope&files=&repos=